frontend/src/features/map_planning/components/toolbar/LayerList.tsx

```diff
@@ -1,109 +1,56 @@
 import useMapStore from '../../store/MapStore';
-import { LayerDto, LayerType } from '@/bindings/definitions';
-import IconButton from '@/components/Button/IconButton';
-import { NamedSlider } from '@/components/Slider/NamedSlider';
-import { ReactComponent as CaretDownIcon } from '@/icons/caret-down.svg';
-import { ReactComponent as CaretRightIcon } from '@/icons/caret-right.svg';
-import { ReactComponent as EyeOffIcon } from '@/icons/eye-off.svg';
-import { ReactComponent as EyeIcon } from '@/icons/eye.svg';
-import { useState } from 'react';
+import { LayerListItem } from './LayerListItem';
+import { LayerDto } from '@/bindings/definitions';
+// import IconButton from '@/components/Button/IconButton';
```

Comment:
Consider adding a short message saying why these imports are disabled, like you did in the component below.